core/templates/dev/head/pages/donate/donate-page.controller.js

```diff
@@ -13,10 +13,11 @@
 // limitations under the License.
 
 /**
- * @fileoverview Controllers for the donate page.
+ * @fileoverview Modules for the donate page.
```

Comment:
This should be controllers - this file has controller not module.

Reply:
Done.

Reply:
done